common-services-api/jdbc/pom.xml

```diff
@@ -56,6 +56,8 @@
         <configuration>
           <instructions>
             <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+            <Bundle-Version>${pentaho-osgi-bundles.version}</Bundle-Version>
+            <Export-Package>org.pentaho.big.data.api.jdbc;version=${pentaho-osgi-bundles.version}</Export-Package>
```

Comment:
This is properly split into interface and implementation. We only need to export the interface package.